### ISSUE TITLE
google_compute_instance_group_manager

### ISSUE THREAD

**Issue by burninmedia** [terraform-google-modules/terraform-google-vault#149]
### TL;DR

the resource block in `modules/cluster/main.tf` uses provider = google-beta
Since this is no longer in beta can we remove this code and use the google provider so there is no need for 2 provider blocks?

### Expected behavior

To create the resource using the google provider.

### Observed behavior

I got the below error which didn't lead me to the actual issue. Looking at why the project wasn't declared, lead me to review the tf code and that is where I found the google-beta provider input.

```
module.vault.module.cluster.google_compute_region_instance_group_manager.vault: Creating...
╷
│ Error: Error creating RegionInstanceGroupManager: Post "https://compute.googleapis.com/compute/beta/projects/PROJECT_NAME/regions/REGION_NAME/instanceGroupManagers?alt=json&prettyPrint=false": oauth2: cannot fetch token: 400 Bad Request
│ Response: {
│   "error": "invalid_grant",
│   "error_description": "reauth related error (invalid_rapt)",
│   "error_subtype": "invalid_rapt"
│ }
│
│   with module.vault.module.cluster.google_compute_region_instance_group_manager.vault,
│   on .terraform/modules/vault/modules/cluster/main.tf line 189, in resource "google_compute_region_instance_group_manager" "vault":
│  189: resource "google_compute_region_instance_group_manager" "vault" {
│
╵
ERRO[0018] 1 error occurred:
	* exit status 1
```

### Terraform Configuration

```hcl
Run this module without a google-beta provider block.
```

### Terraform Version

```sh
Terraform v1.1.8
on darwin_amd64
```

### Additional information

It looks like this resource is now in the google provider per the documentation.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_region_instance_group_manager

Can we just remove line 190 from `/modules/vault/modules/cluster/main.tf`?
`provider = google-beta`
